tooling/impl/src/main/java/org/jboss/windup/tooling/ExecutionResultsImpl.java

```diff
@@ -33,37 +41,67 @@
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
  * @author <a href="mailto:hotmana76@gmail.com">Marek Novotny</a>
  */
+@XmlRootElement(name = "execution-results")
 public class ExecutionResultsImpl implements ExecutionResults
 {
```

Comment:
Isn't this class mixing data and logic too much? Maybe methods working with GraphContext could be somewhere else or at least static?